auth/src/main/java/io/grpc/auth/MoreCallCredentials.java

```diff
@@ -23,10 +23,12 @@
 /**
  * A utility class that converts other types of credentials to {@link CallCredentials}.
  */
-@ExperimentalApi("https//github.com/grpc/grpc-java/issues/1914")
 public final class MoreCallCredentials {
   /**
    * Converts a Google Auth Library {@link Credentials} to {@link CallCredentials}.
+   *
+   * <p>Although this is a stable API, note that the returned instance's API is not stable. See
+   * {@link CallCredentials}.
```

Comment:
Technically the type `CallCredentials` itself is now stable, but just none of the methods on it are.